test/parallel/test-fs-access.js

```diff
@@ -5,6 +5,9 @@
 // and the errors thrown from these APIs include the desired properties
 
 const common = require('../common');
+if (!common.isWindows && process.getuid() === 0)
+  common.skip('as this test should not be run as `root`');
```

Comment:
Totally optional suggestion:

```suggestion
  common.skip('test needs to be run as an unprivileged user');
```

Reply:
Thanks, I like your suggestion and I'm happy to change this. But perhaps this should be done in a follow up PR as there are a couple of these skips that I've added in the past with this wording and changing them all would make the output more consistent. Does that sound alright?

Reply:
Sure, that sounds fine. It was a totally optional suggestion anyway and I wouldn't mind if you ignored it completely. Do it now, do it later, don't do it at all, it's all good by me.